### PR TITLE
docs: add animated image at the top of the README

### PR DESCRIPTION
**Pull request by NathanTarbert**
## Checklist

- [ ] Title matches [Winglang's style guide](https://www.winglang.io/contributing/start-here/pull_requests#how-are-pull-request-titles-formatted)
- [ ] Description explains motivation and solution
- [ ] Tests added (always)
- [ ] Docs updated (only required for features)
- [ ] Added `pr/e2e-full` label if this feature requires end-to-end testing

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Wing Cloud Contribution License](https://github.com/winglang/wing/blob/main/CONTRIBUTION_LICENSE.md)*.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
+
+
+ 
+<div align="center">
+<img width="800"  alt="notification-center-912bb96e009fb3a69bafec23bcde00b0" src="https://github.com/winglang/wing/assets/66887028/6023ceab-060e-4b48-be98-00a546e8c724" alt-text="GIF of Wing's Clouds">
```
Comment (MarkMcCulloh) [inline README.md:5]: `alt` is the attribute for alt text afaik

```suggestion
<img width="800" src="https://github.com/winglang/wing/assets/66887028/6023ceab-060e-4b48-be98-00a546e8c724" alt="GIF of Wing's Clouds">
```

Comment (MarkMcCulloh) [inline README.md:5]: nit: indent

```suggestion
<div align="center">
  <img width="800"  alt="notification-center-912bb96e009fb3a69bafec23bcde00b0" src="https://github.com/winglang/wing/assets/66887028/6023ceab-060e-4b48-be98-00a546e8c724" alt-text="GIF of Wing's Clouds">
```

Comment (MarkMcCulloh) [inline README.md:5]: This gif is a little bit grainy, is there any way it could be less compressed?

Comment (MarkMcCulloh) [inline README.md:5]: I think the image should exist in the repo itself and the src points to it, like the demo.gif below

**Review by MarkMcCulloh — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (NathanTarbert) [reply README.md:5]: That's a good question, I had to reduce the size so it's under 10 MB. 
I can see if @amielpb can reduce the size but keep it hi-rez.

Comment (NathanTarbert) [reply README.md:5]: Ok, that's no problem.

**Comment by NathanTarbert** [comment]
Sorry for the delay. The GIF was added to the logo folder

File: README.md [hunk 2]
```diff
@@ -1,50 +1,56 @@
...
+<a href="https://www.winglang.io/docs/category/faq">FAQ</a>
+▪︎
+<a href="https://www.winglang.io/contributing/status#roadmap">Roadmap</a>
+▪︎
+<a href="https://github.com/winglang/wing/issues">Issues</a>
+▪︎
+<a href="https://github.com/winglang/wing/discussions">Discussions</a>
+▪︎
+<a href="https://www.winglang.io/contributing/">Contribute</a>
+
```
Comment (MarkMcCulloh) [inline README.md:23]: I'm not sure why but it looks like this broke the centering so now these are left-aligned. Was that intentional?

File: apps/wing/logo/banner.gif
Comment (MarkMcCulloh) [inline apps/wing/logo/banner.gif:1]: I'm not sure if it was intentional, but the background of this image looks like static and it's a bit distracting. Especially to the left and right of the text where there's a distortion that looks like [this](https://en.wikipedia.org/wiki/Moir%C3%A9_pattern).
The static on the bottom of the image also isn't moving like the rest.

File: README.md (hunk 2 above)
Comment (MarkMcCulloh) [reply README.md:23]: image

Comment (NathanTarbert) [reply README.md:23]: Hmm, the preview in VS code didn't look weird. I'll check it out

File: apps/wing/logo/banner.gif
Comment (NathanTarbert) [reply apps/wing/logo/banner.gif:1]: I'll see what Amiel can do

Comment (NathanTarbert) [reply apps/wing/logo/banner.gif:1]: @MarkMcCulloh, @amielpb made a new image?

**Review by ShaiBer — APPROVED** [review]

**Review by MarkMcCulloh — APPROVED** [review]

**Pull request merged**